ZenPacks/zenoss/OpenStackInfrastructure/zenpack.yaml

```diff
@@ -13,6 +13,9 @@ class_relationships:
 # Containing Relations
 - Endpoint(components) 1:MC OpenstackComponent
 - Instance 1:MC Vnic
+# Cinder
+- Volume 1:MC Snapshot
```

Comment:
Are you positive that snapshots can not and will never move between volumes?

Containing relationships are dangerous  - I usually avoid them completely unless they are necessary because the object has no unique ID without its parent (which may be the case here, I don't know)